### PR TITLE
Update machine-learning.md

### PR DESCRIPTION
**Pull request by dA505819**
Added "Survival Analysis in R".

### CODE REVIEW

File: details/julia-python-and-r/machine-learning.md [hunk 1]
```diff
@@ -83,13 +83,14 @@
 - [Concrete Compressive Strength Prediction using Machine Learning](https://www.linkedin.com/posts/towards-data-science_concrete-compressive-strength-prediction-activity-6644946997524725760-bbxh)
 - [Learning from unlabelled data](https://www.linkedin.com/posts/montrealai_deeplearning-machinelearning-semisupervisedlearning-activity-6641340729899700224-PP8Q)
 - [Its becoming clearer than ever before that there are huge advantages of training very large models.](https://www.linkedin.com/posts/adamgrzywaczewski_its-becoming-clearer-than-ever-before-that-activity-6645674351192522752-vxsF)
+- [Survival Analysis in R](https://www.geeksforgeeks.org/survival-analysis-in-r/)
```
Comment (neomatrix369) [inline details/julia-python-and-r/machine-learning.md:86]: 👍 
Aaar...rrrrr

**Comment by neomatrix369** [comment]
Thanks Dhruv! Shining like the pole star ;)

**Comment by dA505819** [comment]
@neomatrix369 Thankyou so much!:)
Just on the belief "We rise by lifting others" as knowledge is free for all in the internet era.

**Comment by neomatrix369** [comment]
> @neomatrix369 Thankyou so much!:)
> Just on the belief "We rise by lifting others" as knowledge is free for all in the internet era.

So totally true, one of the key reasons for this repo is to help everyone rise while we are growing and rising. Thankful to everyone who has been contributing. 😍

**Pull request merged**